docs/WRAPPER.md

```diff
@@ -14,18 +14,22 @@
 ```javascript
 const aragon = new Aragon('0xdeadbeef')
 
-// Initialises the wrapper and logs the installed apps
-aragon.init(() => {
-  aragon.apps.subscribe(
-    (apps) => console.log(apps)
-  )
-})
+// Initialises the wrapper
+aragon.init(["0xbeefdead", "0xbeefbeef"], {withAccounts: false}})
 ```
 
 ## init
 
 Initialise the wrapper.
 
+### **Parameters**
+
+-   `accounts` **[Array](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Array)&lt;string>**
+      An optional array of accounts that the user controls
+-   `options` **[Object](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Object)** 
+      An optional options object that contains 'withAccounts' that specifies whether or not we should fetch accounts from the Web3 instance.
+      Ex: ```aragon.init(["0xbeefbeef"], { withAccounts: true })```
```

Comment:
This example doesn't make sense with the way you have it set up. Currently, it will _only_ pull accounts if the account param is null, it might make sense to have the accounts param and the web3 accounts combined if it's called in the way shown in this example.

Reply:
The two can't be combined into a single parameter because there may be times where we'd like to not provide an 
account list but still not fetch accounts from web3.

Given that users should know if they'd like to fetch accounts or provide their own by the time they init, a breaking API change would be to turn the entire argument list into a single options object with `accounts` and `fetchAccountsFromConnectedWeb3` options. If both are provided, the web3 accounts could be concatenated with the provided accounts.